index.html

```diff
@@ -208,16 +208,27 @@ <h5>Get started</h5>
     </div>
 
     <ul class="form-name-box">
-      <li class="form-name-gray">Your name</li>
-      <li class="fill-name">Anastasiia</li>
+      <li class="form-name-gray"><label for="name">Your name</label></li>
+      <li class="fill-name">
+        <input type="text" name="name" id="name" value="Anastasiia" placeholder="Your name" maxlength="30" required>
```

Comment:
- [ ] Kindly remove the set value (`Anastasiia`) of  `value` property for better user experience.

Reply:
Good job in implementing the changes 👍